_includes/partners/contact.html

```diff
@@ -0,0 +1,42 @@
+<main id="main-content">
+  <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  </head>
+  <section>
+    <div class="container partners-container">
+      <div class="grid-row">
+        <div class="tablet:grid-col-9 accordion-container">
+          <script charset="utf-8" type="text/javascript" src="https://js.hsforms.net/forms/shell.js"></script>
+          <script type="text/javascript">
+            hbspt.forms.create({
+              region: "na1",
+              portalId: "{{ site.partnership_contact_portal_id }}",
+              formId: "{{ site.partnership_contact_form_id }}",
+              onFormReady: function($form) {
+                const iFrame = document.getElementById('hs-form-iframe-0');
+                iFrame.setAttribute('title', 'Login dot gov contact us form');
+              },
+            });
+          </script>
+        </div>
+        <div class="tablet:grid-col-3 search-container">
+          <div>
+            {% include search.html size="small" id="header-nav" %}
```

Comment:
The search button looks off here, with inconsistent padding on the right-hand side:

![image](https://user-images.githubusercontent.com/1779930/158176948-83d38120-33e2-49e3-91cb-80aef389e60c.png)

Reply:
Addressed in PR - https://github.com/18F/identity-site/pull/840

Reply:
This doesn't appear to be fixed in #840, the search button is still squished.

![image](https://user-images.githubusercontent.com/1779930/161979870-a4076f68-fe33-462e-ad41-649a2172b990.png)

I think we can probably remove `size` here, since the "Small" / "Default" sizes were normalized in the design system v6.3.0.

```suggestion
            {% include search.html id="header-nav" %}
```

I haven't tested, but we could probably update the other one as well to avoid similar issues:

https://github.com/18F/identity-site/blob/beccd3d333cccfc91e8579a447c28dc2eff6db1f/_layouts/partners/faq.html#L163